### ISSUE TITLE
`SELECT` should have more relationships

### ISSUE THREAD

**Issue by robartsd** [ArnoldSmith86/virtualtabletop#586]
Currently we can SELECT with relationships directly comparing a property to a value. We can also select based on a property matching an element of an array. It is difficult to select based on a value being in an array contained in a property.

Proposed new relationships:
 - `contains` : selects widgets where the property matches the value or holds an array that has an element that matches the value
 - `not in` : boolean inverse of `in`
 - `does not contain`: boolean inverse of contains

**Comment by RaphaelAlvez** [comment]
I believe the contains was added in "in". Take a look at the JSON help PR code. The "in" added was a mix off arrays' in and strings' in

So for this issue I would only suggest adding "!in" (and "!contains" and "contains" if you think I'm mistaken)

**Comment by larkob** [comment]
@robartsd Could you please clarify how the `contains` is meant to work? SELECT is performed on an array of objects, so do you mean you want to search the array of widgets for a widget that has a property with a matching value? I believe your `contains` is what we currently have as `includes`, right?

**Comment by larkob** [comment]
Please check out the PR test room (in #615) for an example in which the SELECT picks a card with a property that has an array with a specific value

**Comment by robartsd** [comment]
Yes I believe you are correct about `includes`. I'll test later.
